bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -13,18 +14,21 @@
 from bot.bot import Bot
 from bot.constants import Bot as BotConfig, Channels, Emojis, Guild, MODERATION_ROLES, Roles, STAFF_ROLES
 from bot.converters import MemberOrUser, UnambiguousMemberOrUser
-from bot.exts.recruitment.talentpool._review import Reviewer
+from bot.exts.recruitment.talentpool._api import Nomination, NominationAPI
+from bot.exts.recruitment.talentpool._review import RECENT_ACTIVITY_DAYS, Reviewer
 from bot.log import get_logger
 from bot.pagination import LinePaginator
 from bot.utils import time
 from bot.utils.channel import get_or_fetch_channel
 from bot.utils.members import get_or_fetch_member
 
-from ._api import Nomination, NominationAPI
-
 AUTOREVIEW_ENABLED_KEY = "autoreview_enabled"
 REASON_MAX_CHARS = 1000
 
+# The number of days that a user can have no activity (no messages sent)
+# until they should be removed from the talentpool.
+DAYS_UNTIL_INACTIVE = 30
```

Comment:
Wouldn't it be a better idea to move these to their own section maybe in the config file ?

Reply:
Yeah, i'm not sure what our general policy on this should be now. I know we wanted to try and keep the config file fairly concise and remove unnecessary things, although making durations easily configurable for development would probably be helpful.

Reply:
@shtlrs do you think it would be easier to do this now, or in a new PR once #2408 is merged to avoid conflicts? I'm happy to do either.

Reply:
I think it's fine to leave as is for now, we also count on waiting for @mbaruh's filters PR to get done so that we could merge.
I would then prune all the filter constants, and try to group the rest by context as  well.

Reply:
(my PR isn't blocking, by all means keep the constants there and I'll just resolve the conflict by deleting them)